autohooks/settings.py

```diff
@@ -70,9 +70,9 @@ def write(self, filename: Path) -> None:
         overridden.
         """
         if filename.exists():
-            toml_doc: Union[
-                tomlkit.toml_document.TOMLDocument, tomlkit.items.Table
-            ] = tomlkit.loads(filename.read_text())
+            toml_doc: tomlkit.toml_document.TOMLDocument = tomlkit.loads(
```

Comment:
You can shorten the typing by using the main module
```suggestion
            toml_doc: tomlkit.TOMLDocument = tomlkit.loads(
```